### PR TITLE
Adding manual config

### PR DESCRIPTION
**Pull request by Simplychee**
https://deploy-preview-159--shiny-alfajores-ede8d8.netlify.app/docs/shipping/AWS/AWS-EC2

~https://deploy-preview-159--shiny-alfajores-ede8d8.netlify.app/docs/shipping/compute/winlogbeat/~

https://deploy-preview-159--shiny-alfajores-ede8d8.netlify.app/docs/shipping/Operating-Systems/Windows

https://deploy-preview-159--shiny-alfajores-ede8d8.netlify.app/docs/shipping/other/linux-data/

https://deploy-preview-159--shiny-alfajores-ede8d8.netlify.app/docs/shipping/Compute/localhost-mac

### CODE REVIEW

File: docs/shipping/AWS/aws-ec2.md [hunk 1]
```diff
@@ -41,7 +41,172 @@ This integration creates a Kinesis Data Firehose delivery stream that links to y
 
 
 
-## Configure AWS to forward metrics to Logz.io
+## Manually configure OpenTelemetry on Linux
+
```
Comment (ShiranAvidov) [inline docs/shipping/AWS/aws-ec2.md:45]: Send your AWS EC2 logs and metrics using OpenTelemetry collector service

File: docs/shipping/AWS/aws-ec2.md [hunk 2]
```diff
@@ -41,7 +41,172 @@ This integration creates a Kinesis Data Firehose delivery stream that links to y
```
Comment (ShiranAvidov) [inline docs/shipping/AWS/aws-ec2.md:42]: The whole beginning should be in the cloudwatch metrics section

File: docs/shipping/Compute/winlogbeat.md [hunk 3]
```diff
@@ -20,7 +20,147 @@ drop_filter: []
 [Winlogbeat 8](https://www.elastic.co/guide/en/beats/winlogbeat/8.7/winlogbeat-installation-configuration.html#installation), [Winlogbeat 7](https://www.elastic.co/guide/en/beats/winlogbeat/7.x/winlogbeat-installation-configuration.html#installation), or [Winlogbeat 6](https://www.elastic.co/guide/en/beats/winlogbeat/6.8/winlogbeat-installation.html).
 
  
+## Manually configure OpenTelemetry on Localhost
```
Comment (ShiranAvidov) [inline docs/shipping/Compute/winlogbeat.md:23]: Not really sure how the agent is relevant in Winlogbeat... they also don't need Winlogbeat to send logs if they use the agent...

File: docs/shipping/Other/linux.md [hunk 4]
```diff
@@ -20,7 +20,162 @@ drop_filter: []
 * Root access
 * Port 5000 open
```
Comment (ShiranAvidov) [inline docs/shipping/Other/linux.md:21]: you don't need port 5000 open if you use the agent

File: docs/shipping/Other/linux.md [hunk 5]
```diff
@@ -20,7 +20,162 @@ drop_filter: []
...
 
+
+## Manually configure OpenTelemetry on Localhost
+
```
Comment (ShiranAvidov) [inline docs/shipping/Other/linux.md:25]: Send your Linux machine logs and metrics using OpenTelemetry service

File: docs/shipping/Compute/localhost-mac.md [hunk 6]
```diff
@@ -0,0 +1,173 @@
...
+logs2metrics: []
+metrics_dashboards: ['2gsQP2xRef7dkwt8pxWieo']
+metrics_alerts: ['hWld33IEO6gZMpp2e4vs0']
+drop_filter: []
+---
+
+
+
+## Manually configure OpenTelemetry on Mac
+
```
Comment (ShiranAvidov) [inline docs/shipping/Compute/localhost-mac.md:20]: Send your Mac machine logs and metrics using Opentelemetry service

**Review by ShiranAvidov — CHANGES_REQUESTED** [review]

File: docs/shipping/Compute/winlogbeat.md (hunk 3 above)
Comment (Simplychee) [reply docs/shipping/Compute/winlogbeat.md:23]: So should I just remove it? I thought this is the "Windows localhost" page? 🤔  @ShiranAvidov

**Review by mirii1994 — CHANGES_REQUESTED** [review]

**Review by mirii1994 — APPROVED** [review]

**Pull request merged**